concent_api/middleman/utils.py

```diff
@@ -0,0 +1,78 @@
+from typing import Dict
+
+import asyncio
+
+from logging import getLogger
+from logging import Logger
+
+from middleman.constants import ResponseQueueItem
+
+STANDARD_ERROR_MESSAGE = "This connection has been already added"
```

Comment:
How about moving this to constants too ?

Reply:
it is used only here, but I might if you prefer it that way

Reply:
done